hi-ml-multimodal/src/health_multimodal/image/model/__init__.py

```diff
@@ -3,13 +3,15 @@
 #  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 #  -------------------------------------------------------------------------------------------
 
+from .model import BaseImageModel
 from .model import ImageModel
 from .model import ResnetType
 from .model import get_biovil_resnet
 from .model import CXR_BERT_COMMIT_TAG
 from .model import BIOMED_VLP_CXR_BERT_SPECIALIZED
 
 __all__ = [
+    "BaseImageModel",
```

Comment:
We might also want to add `ImageModelInput` here, any code that subclasses `BaseImageModel` will need to access this class.

Reply:
[Resolving as we've changed method and this is now outdated]